lib/ui/platform_dispatcher.dart

```diff
@@ -185,6 +185,15 @@ class PlatformDispatcher {
   /// otherwise.
   FlutterView? view({required int id}) => _views[id];
 
+  /// The ID of the currently focused [FlutterView].
+  ///
+  /// Returns -1 if no view is focused.
```

Comment:
This API suffers from "a tree falls in the forest" problem, in that its value may change but whoever read it still has the old value. However, together with the `onViewFocusChange`, you get the full story. Therefore, I think `focusedViewId` and `onViewFocusChange` should cross-link into each other from their respective dartdocs.

Reply:
+1 to cross-linking :)